timesketch/frontend-ng/src/views/Story.vue

```diff
@@ -560,4 +614,4 @@ export default {
 .theme--dark.v-application code {
   background-color: transparent !important;
 }
-</style>
+</style>
```

Comment:
Keep the empty line at the end.

```suggestion
</style>

```